routes/web.php

```diff
@@ -15,12 +15,16 @@
 
 
 
-Route::get('data_input', function () {
-    return view('homepage.data_input');
+Route::get('index', function () {
+    return view('activity.index');
 });
 
-Route::get('fulfilled', function () {
-    return view('homepage.fulfilled');
+Route::get('create', function () {
+    return view('activity.create');
+});
+
+Route::get('activitySelection', function () {
```

Comment:
subsequently, you could do 
`Route::get('/activity/selection', function() {
return view('link.to.the.view');})->name('activity.selection);`

That way you could use it as 
`{{ route('activity.selection') }} or {{ url('activity/selection') }}`


Don't use camel case of underscore and things of such to name routes.

Reply:
ohh , noted . Thanks